### ISSUE TITLE
[BUG] title on check out page returns to mybigcommerce Storefronts page

### ISSUE THREAD

**Issue by buty619** [vercel/commerce#27]
## Description

when i'm on the checkout page from mybigcommerce if i click the title of the page they re directs me to the Storefronts page from mybigcommerce, how i can change that url so only re directs to nextjs commerce page ?

### Check Out Page

<img width="1440" alt="Screen Shot 2020-10-27 at 9 21 19 PM" src="https://user-images.githubusercontent.com/36422189/97382982-87f5df80-189a-11eb-9deb-2aad0c4680f1.png">

### Redirects page

<img width="1438" alt="Screen Shot 2020-10-27 at 9 21 46 PM" src="https://user-images.githubusercontent.com/36422189/97382988-8af0d000-189a-11eb-9ebb-11f2fa310e54.png">

**Comment by okbel** [comment]
We'll send this to BigCommerce right now. Thank you! I'll keep you posted.

**Comment by RayeEThompson** [comment]
TY @okbel and @buty619, I am looking into this now might need a day to confirm my suspicions. Are you in our BigCommerce Dev Slack? If you can join by filling out this [form](https://forms.gle/K9MWsQrxciGbXSeBA). I can see if there is anyone having this issue in that forum as well.

**Comment by buty619** [comment]
@RayeEThompson  thanks for time investigating this, i gonna fill the form and see if there is anything i can do to help the team!! if you need some help pls let me know.

**Comment by bryant273092** [comment]
Any update on this issue? I tried changing the link with a custom script but that did not work.

**Comment by syedcts** [comment]
Hi @okbel @RayeEThompson, Any update on this? I think We all are facing this issue.

**Comment by syedcts** [comment]
Hi @buty619 @okbel,
I had a word with BC support, they said @RayeEThompson is not with Bigcommerce anymore. They suggested a few things and with that help, I was able to fix this issue and also made some more changes to the checkout and checkout success page.
To achieve this in headless, BC provides an option to make changes in these pages via the stencil theme edit option, I basically hardcoded my home URL on the checkout.html page, although we can do this by adding that URL in the config file and import via Handlebar. 
Also, I made some text changes(By changing in translation file), and hide the signup/login option from the checkout and checkout-success page with the help of CSS provided in the default theme.
these changes will work fine in projects based on this repo also.
@okbel I think there are some more GH open issues related to this. I think we can link them and close them.
